jmclient/jmclient/payjoin.py

```diff
@@ -76,7 +76,8 @@ class JMPayjoinManager(object):
     pj_state = JM_PJ_NONE
 
     def __init__(self, wallet_service, mixdepth, destination,
-                 amount, server, disable_output_substitution=False):
+                 amount, server, disable_output_substitution=False,
+                 mode="command-line"):
```

Comment:
Why can't we just use `oneshot=True` boolean here instead?

Reply:
I guess I just wanted to cover the possibility there were more than 2.

Reply:
But can we think about any? I mean - this basically handles that oneshot true/false thing, that's all.

Reply:
Doesn't seem worth changing for that reason.

Reply:
Ok, agree, not so important.